src/SUMMARY.md

```diff
@@ -40,10 +40,43 @@
 
 # Day 1: Afternoon
 
+- [Control Flow](control-flow.md)
+  - [Blocks](control-flow/blocks.md)
+  - [if expressions](control-flow/if-expressions.md)
+  - [for expressions](control-flow/for-expressions.md)
+  - [while expressions](control-flow/while-expressions.md)
+  - [break & continue](control-flow/break-continue.md)
+  - [loop expressions](control-flow/loop-expressions.md)
+
 - [Variables](basic-syntax/variables.md)
   - [Type Inference](basic-syntax/type-inference.md)
   - [static & const](basic-syntax/static-and-const.md))
   - [Scopes and Shadowing](basic-syntax/scopes-shadowing.md)
+- [Enums](enums.md)
+  - [Variant Payloads](enums/variant-payloads.md)
+  - [Enum Sizes](enums/sizes.md)
+
+- [Novel Control Flow](advanced-control-flow.md)
```

Comment:
I like this distinction -- "control flow you've heard of" vs. "new forms of control flow" (that they're not really _new_, but fall under the "modern language features" part of the "Why Rust" slide).